cloudferrylib/scheduler/scenario.py

```diff
@@ -32,13 +26,22 @@ class Scenario(object):
     def __init__(self, path_tasks, path_scenario):
         self.path_tasks = path_tasks
         self.path_scenario = path_scenario
+        self.tasks = None
+        self.namespace = None
+        self.migration = None
+        self.preparation = None
+        self.rollback = None
 
-    def init_tasks(self, init={}):
+    def init_tasks(self, init):
         with open(self.path_tasks) as tasks_file:
             tasks_file = yaml.load(tasks_file)
-            actions = {}
-            for mod in tasks_file['paths']:
-                actions.update(self.get_actions(mod))
+            # actions = {}
+            # for mod in tasks_file['paths']:
+            #     actions.update(self.get_actions(mod))
+
+            actions = dict(extensions.available_extensions(
```

Comment:
Ah, I see, I'm not right. Maybe return dict there?.. Type conversion to dict looks odd.